Source/WebKit/WebProcess/GPU/webrtc/RemoteVideoFrameObjectHeapProxyProcessor.cpp

```diff
@@ -150,18 +150,18 @@ void RemoteVideoFrameObjectHeapProxyProcessor::newConvertedVideoFrameBuffer(std:
 
 RefPtr<NativeImage> RemoteVideoFrameObjectHeapProxyProcessor::getNativeImage(const WebCore::VideoFrame& videoFrame)
 {
-    auto& connection = WebProcess::singleton().ensureGPUProcessConnection().connection();
+    Ref connection = WebProcess::singleton().ensureGPUProcessConnection().protectedConnection();
```

Comment:
You don't need BOTH `Ref` and `protectedConnection()`. Just this is enough:
```
Ref connection = WebProcess::singleton().ensureGPUProcessConnection().connection();
```

Reply:
Good point-fixed!